lib/data.rb

```diff
@@ -0,0 +1,67 @@
+require './lib/dmv'
+require './lib/facility'
+require './lib/vehicle'
+require './lib/dmv_data_service'
+require 'json'
+
+class Create
+attr_reader :facilities, :vehicles
+    def initialize
+        @facilities = []
+        @vehicles =[]
+    end
+
+    def create_facility(facility_list)
+    
+        facility_list.each do|facility|
+            keys =  facility.keys
+
+
+
+            keys.each do |key|
+                if key.to_s.include?('address') ||  key.to_s.include?('location') 
+                        if facility.has_key?(:location_1) ==true 
+                            hash = JSON.parse(facility.dig(:location_1, :human_address))
+                            facility.update({:address => "#{hash["address"]}, #{hash["city"]}, #{hash["state"]} #{hash["zip"]}"})
+                        else
+                            facility.update({:address => facility.fetch(key)})
+                        end 
+                elsif   key.to_s.include?('title') ||  key.to_s.include?('name') 
+                    facility.update ({:name => facility.fetch(key)})
+        
+                elsif   key.to_s.include?('phone') 
+                        facility.update({:phone => facility.fetch(key)}) 
+                end
```

Comment:
Yeah, I like the idea of creating a method like this where we can input any facility list and if the keys are different, we just add that new key into this method.